src/Akka.Streams.Kafka/Stages/ConsumerStage.cs

```diff
@@ -170,5 +169,36 @@ private void PullQueue()
         }
 
         protected override void OnTimer(object timerKey) => PullQueue();
+        
+        private bool IsBrokerErrorRetriable(Error error)
+        {
+            switch (error.Code)
```

Comment:
Regarding to https://cwiki.apache.org/confluence/display/KAFKA/A+Guide+To+The+Kafka+Protocol#AGuideToTheKafkaProtocol-ErrorCodes